var/spack/repos/builtin/packages/py-msal/package.py

```diff
@@ -22,6 +22,7 @@ class PyMsal(PythonPackage):
     version("1.3.0", sha256="5442a3a9d006506e653d3c4daff40538bdf067bf07b6b73b32d1b231d5e77a92")
     version("1.0.0", sha256="ecbe3f5ac77facad16abf08eb9d8562af3bc7184be5d4d90c9ef4db5bde26340")
 
+    # https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.20.0/setup.py
```

Comment:
Up to you, but I don't usually include URLs for stuff like this. It will quickly become out of date as it only applies to the 1.20.0. I usually just add a comment like `setup.py`, `pyproject.toml`, etc. so I know which file to check in, but allow the file to be release-specific. I don't actually check GitHub personally, I just download from PyPI and look through the source code.

Reply:
I prefer to leave this here because it makes it easier to check, the user can change the tag in the URL.